### ISSUE TITLE
Static can be used for variables not only constants/functions

### ISSUE THREAD

**Issue by sancarn** [serkonda7/vscode-vba#85]
### Description and Reproduction Steps

https://github.com/serkonda7/vscode-vba/blob/main/syntaxes/vba.tmGrammar.yml#L49

This doesn't account for the following:

```vb
Sub main()
  static myObj as object: if myObj is nothing then set myObj = something()
  '...
End Sub
```

### Current Behavior

`Static` keyword not highlighted

### Expected Behavior

`Static` key word highlighted.

### Additional Information

Might have to include `static` keyword here?

https://github.com/serkonda7/vscode-vba/blob/main/syntaxes/vba.tmGrammar.yml#L73

### Environment details

OS:
VS Code Version:
Extension Version:

**Comment by serkonda7** [comment]
Good catch. I will fix it.

Also a link to reference: https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/static-statement

**Comment by serkonda7** [comment]
Closed in [93d1791](https://github.com/serkonda7/vscode-vba/commit/93d1791ba4013d924750f7890fda9394d1c6dccd)
